Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -348,7 +348,8 @@ extension BrowserViewController {
             UIKeyCommand(action: #selector(findInPageAgainKeyCommand), input: "g", modifierFlags: .command, discoverabilityTitle: shortcuts.FindAgain),
 
             // View
-            UIKeyCommand(action: #selector(zoomIn), input: "+", modifierFlags: .command, discoverabilityTitle: shortcuts.ZoomIn),
+            UIKeyCommand(action: #selector(zoomIn), input: "=", modifierFlags: .command, discoverabilityTitle: shortcuts.ZoomIn),
+            UIKeyCommand(action: #selector(zoomIn), input: "+", modifierFlags: .command),
```

Comment:
I think we should put the discoverabilityTitle on the command + keyboard shortcuts, since it's the "official" way of zooming in.

Reply:
yea, my bad on that one.